packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -170,6 +154,23 @@ def check_adminrouter_metrics():
         check_adminrouter_metrics()
 
 
+def test_metrics_diagnostics(dcos_api_session):
+    """Assert that DC/OS Diagnostics metrics on master are present."""
+    nodes = get_master_and_agents(dcos_api_session)
+
+    for node in nodes:
+        @retrying.retry(wait_fixed=STD_INTERVAL, stop_max_delay=METRICS_WAITTIME)
+        def check_diagnostics_metrics():
+            response = get_metrics_prom(dcos_api_session, node)
+            for family in text_string_to_metric_families(response.text):
+                for sample in family.samples:
+                    if sample[0].startswith('bundle_creation_time_seconds'):
```

Comment:
if the metric you're looking for is just called `bundle_creation_time_seconds`, might be better to assert with `==` instead of `startswith`